setup.py

```diff
@@ -78,6 +80,7 @@
     'doc': ['sphinx', 'sphinx_rtd_theme', 'sphinxcontrib-apidoc'],
     # Other
     'geoviews': ['geoviews'],
+    'hvplot': ['hvplot'],
```

Comment:
```suggestion
    'hvplot': ['hvplot', 'geoviews', 'cartopy'],
```
Running `scn.to_hvplot()` without installing `geoviews` and `cartopy` resulted in `ImportError: Geographic projection support requires GeoViews and cartopy.`, so they need to be added as requirements for `hvplot` feature.